views/index.ejs

```diff
@@ -0,0 +1,47 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="/stylesheets/style.css">
+    <title>Coffee and Books</title>
+</head>
+<body>
+        
+
+    Add Location
+    <form action="/create" method="post">
+        <label for="name">Name</label>
+        <input type="text" name="name">
+        <label for="longitude">Longitude</label>
+        <input type="text" name="longitude">
```

Comment:
Do you expect the user to know the coordinates of a place? You need to fill this fields when the user clicks the map. You can avoid the user to change this with the attribute: readonly="readonly".

Of course, you will need to improve your javascript to retrieve the latitude and longitude from the map. Search info about: navigator.geolocation.getCurrentPosition().